src/api/auth.rs

```diff
@@ -88,7 +88,8 @@ pub async fn code(conn: DbConn, code: &str, cookies: &CookieJar<'_>) -> Result<R
                     // Create the user's personal team
                     let team = diesel::insert_into(teams)
                         .values(&NewTeam {
-                            name: format!("{}'s team", info.name),
+                            name: Some(format!("{}'s team", info.name)),
+                            slug: into_slug(&info.name, true),
```

Comment:
Should be based off username, not "real name" field. This should also fix the issue with spaces and such, but we should substitute `_` for `-` in usernames and remove all other characters.